src/language-handlebars/clean.js

```diff
@@ -6,7 +6,7 @@ module.exports = function (ast, newNode) {
 
   // (Glimmer/HTML) ignore TextNode whitespace
   if (ast.type === "TextNode") {
-    const trimmed = ast.chars.trim();
+    const trimmed = ast.chars.trim().replace(/\s+/g, " ");
```

Comment:
Please, only do nessary change, should check attribute name.

Reply:
@fisker actually I don't quite understand why we need to do this. Could you explain? Why are we "cleaning" AST and when does this happen? Also, why does the test fail now if we only changed Prettier output and not the underlying AST? Checking for attribute==class will involve some duplication from Glimmer's printer which I'd rather avoid. I'm also not sure if this place is performance critical?

Reply:
This only runs when comparing the AST during test, so performance is  not very  important, but  make  sure the formated / original code  have the same   AST is very important.

Reply:
> Also, why does the test fail now if we only changed Prettier output and not the underlying AST?

You  changed the  output, it parse to  different ASTs.

Reply:
I'm not  familiar with  glimmer  AST , but I checked failed tests on  CI , the  attribute with  `TextNode` have  `"name": "class"`, can  we  use it ?.